data/bash-completion/fwupdmgr

```diff
@@ -82,6 +83,16 @@ _show_device_ids()
 	COMPREPLY+=( $(compgen -W "${description}" -- "$cur") )
 }
 
+_show_release_versions()
+{
+	if ! command -v jq &> /dev/null; then
+		return 0
+	fi
+	local description
+	description="$(command fwupdmgr get-releases $1 --json 2>/dev/null | jq '.Releases[].Version')"
```

Comment:
`get-releases` is better indeed!

better to double quote the `"$1"` variable.

Reply:
Agree, thanks. fixed.